postgres/test/cds-build.test.js

```diff
@@ -27,4 +27,10 @@ describe('cds build plugin', () => {
     execSync('npx cds build --production', { cwd: workDir })
     expect(fs.existsSync(path.join(dbDest, 'csn.json'))).to.be.true
   })
+
+  test('should retain assert_integrity setting', () => {
+    execSync('CDS_FEATURES_ASSERT__INTEGRITY=db npx cds build --production', { cwd: workDir })
```

Comment:
```suggestion
    execSync('CDS_FEATURES_ASSERT_INTEGRITY=db npx cds build --production', { cwd: workDir })
```
Is the double `_` intentional?

I would prefer to add the `cds.features.assert_integrity` directly into the `tiny-sample/package.json`

Reply:
Yes, the double `_` [is intentional](https://cap.cloud.sap/docs/guides/databases#database-constraints):
![Screenshot 2025-02-19 at 11 08 53](https://github.com/user-attachments/assets/b76e89a4-f2e3-4cf1-bb0f-a56c1ee0f536)

Can gladly add it to the _package.json_, but that this setting is active for all tests. Probably not an issue, just didn't want to intrude in unrelated tests.